core/endpoint.go

```diff
@@ -120,6 +128,37 @@ type Endpoint struct {
 
 	// NumberOfSuccessesInARow is the number of successful evaluations in a row
 	NumberOfSuccessesInARow int `yaml:"-"`
+
+	// SSH is the configuration of SSH monitoring.
+	SSH *SSH `yaml:"ssh,omitempty"`
+}
+
+type SSH struct {
+	// Port is the port to connect to the SSH server.
+	Port string `yaml:"port,omitempty"`
+	// User is the username to use when connecting to the SSH server.
+	User string `yaml:"user,omitempty"`
```

Comment:
I'd rather this be 
```go
Username string `yaml:"username,omitempty"`
```
so that, uh, 
```yaml
ssh:
  username: "root"
  password: "toor"
```
aligns